src/content/guides/how-to-id-scam-tokens/index.md

```diff
@@ -0,0 +1,89 @@
+---
+title: How to identify scam tokens
+description: What are scam tokens, how do they make themselves look legitimate, and how to detect them to avoid the scam.
+lang: en
+---
+
+# How to identify scam tokens
+
+One of the most common uses for Ethereum is for a group to create a tradable token, in a sense their own currency. These tokens typically follow a standard, [ERC-20](/developers/docs/standards/tokens/erc-20/). However, anywhere there are legitimate use cases that bring value, there are also criminals who try to steal that value for themselves. In this case, there are two way in which they are likely to do so:
+
+- Try to sell you *scam tokens*, which look like the legitimate token you want to purchase, but are issued by the scammers and worth nothing.
+- Try to direct you into their own user interface, where they attempt to trick you into signing transactions that give them your ETH, your real tokens, etc. 
+
+To illustrate what scam tokens are, and how to identify them, we are going to look at [`wARB`](https://etherscan.io/token/0xb047c8032b99841713b8e3872f06cf32beb27b82). This token attempts to look like the legitimate [`ARB`](https://etherscan.io/address/0xb50721bcf8d664c30412cfbc6cf7a15145234ad1) token.
+
+<ExpandableCard
+title="What is ARB?"
+contentPreview=''>
+
+[Arbitrum](https://developer.arbitrum.io/intro/) is an organization that develops and manages [optimistic rollups](/developers/docs/scaling/optimistic-rollups/). Initially Arbitrum was organized as a for-profit company, but they recently [started to decentralize](https://arbitrumfoundation.medium.com/arbitrum-the-next-phase-of-decentralization-e7f8b37b5226). As part of that process, they issued a tradeable [governance token](/dao/#token-based-membership).
```

Comment:
This has been an ongoing challenge getting our MDX components (React components within markdown pages) to mix well with nested markdown syntax, and to also be handled appropriately when the content makes it to Crowdin for translation.

The solution we've decided in the meantime has been sticking with `html` while inside a React component within a markdown page. This has been the most reliable at rendering correctly, and being parsed properly by Crowdin.

Even though this one renders okay, we should switch it for Crowdin, and consistency throughout.

```suggestion
<a href="https://developer.arbitrum.io/intro/">Arbitrum</a> is an organization that develops and manages <a href="/developers/docs/scaling/optimistic-rollups/">optimistic rollups</a>. Initially Arbitrum was organized as a for-profit company, but they recently <a href="https://arbitrumfoundation.medium.com/arbitrum-the-next-phase-of-decentralization-e7f8b37b5226">started to decentralize</a>. As part of that process, they issued a tradeable <a href="/dao/#token-based-membership">governance token</a>.
```

Reply:
Also, to help this stand the test of time more, perhaps we change away from "but they recently started to decentralize" and just say "but then took steps to decentralize" (or something similar that removes the relative "recently"